1차 세미나/work.ts

```diff
@@ -0,0 +1,33 @@
+interface member {
```

Comment:
```suggestion
interface Member {
```
인터페이스의 이름은 PascalCase, 프로퍼티 이름은 camelCase 로 하시는 것을 추천드립니다.

Typescript 에서 Interface 의 네이밍 컨벤션은 이 [링크](https://dirask.com/posts/TypeScript-interface-naming-convention-x1RwqD)에서 더 잘 확인하실 수 있습니다.

Reply:
좋은 정보 감사합니다! 바로 수정하겠습니다!